File: quality/tests/fixtures/typescript/sample.ts
// Conformance fixture: known escapes and one function of known complexity.
export const a: any = 1;
// @ts-ignore
export const b = (a as any).c;
const d = a!.e;
/* eslint-disable */
it.skip("x", () => {});
const notAnEscape = "Company"; // "any" inside a word is not an escape

export function branchy(n: number): number {
  if (n === 1) { return 1; }
  if (n === 2) { return 2; }
  if (n === 3) { return 3; }
  if (n === 4) { return 4; }
  return 0;
}

File: quality/tests/fixtures/python/sample.py
"""Conformance fixture: known escapes and one function of known complexity."""
import json  # noqa: F401

x = json.loads("1")  # type: ignore
y = x  # type: ignore[assignment]
# a comment merely mentioning the words type and ignore is not a site


def branchy(a):
    """cyclomatic 5: four ifs and the fall-through."""
    if a == 1:
        return 1
    if a == 2:
        return 2
    if a == 3:
        return 3
    if a == 4:
        return 4
    return 0


def swallow():
    try:
        pass
    except:
        pass
